lib/collector/remote-method.js

```diff
@@ -88,7 +90,32 @@ RemoteMethod.prototype.invoke = function invoke(payload, nrHeaders, callback) {
       if (err) {
         return callback(err)
       }
-      this._post(serialized, nrHeaders, callback)
+      this._post(
+        serialized,
+        nrHeaders,
+        function onResponse(err, res) {
+          if (err) {
+            return callback(err)
+          }
+          if (this._metrics) {
+            const payloadByteLength = byteLength(serialized)
+            const responseByteLength = res.payload ? byteLength(JSON.stringify(res.payload)) : 0
+            this._metrics.measureBytes(
+              `${NAMES.SUPPORTABILITY.NODEJS}/Collector/Output/Bytes`,
```

Comment:
you should probably put all this into a new metric constant in `lib/metrics/names`

Reply:
maybe 2 to account for the top level and then the sub destination